R/RTemplate.R

```diff
@@ -1,3 +1,11 @@
+findFileInDirs = function(file,dirs) {
+  pot = c(file, paste(dirs,file,sep="/"))
```

Comment:
I just noticed that there is a bug here. If `dirs` is `NULL` (or `character(0)`), then `paste(NULL,"file",sep="/")` will return `/file`. There should be a `if (length(dirs) > 0)` here.

Seems it is a bug in the original code.